modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/templates/field-options-toolbar.soy

```diff
@@ -45,5 +45,5 @@
 {template .field_settings_toolbar_item}
 	{@param option: [buttonClass: string, handler: string, label: string]}
 
-	<a class="dropdown-item {$option.buttonClass or ''}" data-handler="{$option.handler}" href="javascript:;" title="{$option.label}">{$option.label}</a>
+	<a class="dropdown-item form-ddm-settings-item {$option.buttonClass or ''}" data-handler="{$option.handler}" href="javascript:;" title="{$option.label}">{$option.label}</a>
```

Comment:
hey @clovisdasilvaneto , I think a better name for that would be: `ddm-form-field-actions-item`